src/sentry/integrations/vsts/notify_action.py

```diff
@@ -1,42 +1,99 @@
 from __future__ import absolute_import
 
 import logging
+import six
 
 from django import forms
 
+from sentry.models.integration import Integration
 from sentry.rules.actions.base import TicketEventAction
+from sentry.shared_integrations.exceptions import IntegrationError
 from sentry.utils.http import absolute_uri
 from sentry.web.decorators import transaction_start
 
+from sentry.integrations.vsts.integration import VstsIntegration
+
 logger = logging.getLogger("sentry.rules")
 
 INTEGRATION_KEY = "integration"
 
 
 class AzureDevopsNotifyServiceForm(forms.Form):
-    # TODO 2.0 Add form fields.
+    vsts_integration = forms.ChoiceField(choices=(), widget=forms.Select())
+
     def __init__(self, *args, **kwargs):
+        integrations = [(i.id, i.name) for i in kwargs.pop("integrations")]
         super(AzureDevopsNotifyServiceForm, self).__init__(*args, **kwargs)
 
+        if integrations:
+            self.fields["vsts_integration"].initial = integrations[0][0]
+
+        self.fields["vsts_integration"].choices = integrations
+        self.fields["vsts_integration"].widget.choices = self.fields["vsts_integration"].choices
+
     def clean(self):
         return super(AzureDevopsNotifyServiceForm, self).clean()
 
 
 class AzureDevopsCreateTicketAction(TicketEventAction):
     form_cls = AzureDevopsNotifyServiceForm
-    label = u"TODO Create an Azure DevOps work item in {name} with these "
+
     ticket_type = "an Azure DevOps work item"
     link = "https://docs.sentry.io/product/integrations/azure-devops/#issue-sync"
+    label = u"Create an Azure DevOps work item in {vsts_integration} with these "
+    prompt = "Create an Azure DevOps work item"
     provider = "vsts"
     integration_key = INTEGRATION_KEY
 
     def __init__(self, *args, **kwargs):
         super(AzureDevopsCreateTicketAction, self).__init__(*args, **kwargs)
-        # TODO 2.1 Add form_fields
-        self.form_fields = {}
+        integration_choices = [(i.id, i.name) for i in self.get_integrations()]
+
+        if not self.get_integration_id() and integration_choices:
+            self.data[self.integration_key] = integration_choices[0][0]
+
+        self.form_fields = {
+            "vsts_integration": {
```

Comment:
@ceorourke once you rename `vsts_integration` to `integration`, I think you can move this functionality to the `TicketEventAction` class since it will be the same as Jira